lib/search_vector.rb

```diff
@@ -7,8 +7,9 @@ module SearchVector
   def self.query(query_string, expr=nil)
     (expr || Qexpr.new).where(
       "(supporters.fts @@ websearch_to_tsquery('english', $search)
-      OR donations.fts  @@ websearch_to_tsquery('english', $search))",
-      { search: query_string }
+      OR donations.fts  @@ websearch_to_tsquery('english', $search)
+      OR payments.id::varchar(255) = $search)",
```

Comment:
Hmmm, this worries me. By converting to varchar, it will likely bypass the index on the id column and then lead to running a search that has to go through every possible payment. Can you run an a query analysis of this query, i.e. call the query in the dbconsole but add `EXPLAIN ANALYZE` at the beginning?

Reply:
It became really slow, I didn't realize that. I did a workaround to prevent that.